pkg/internal/infraolly/process/snapshot_test.go

```diff
@@ -218,3 +218,17 @@ func Test_usernameFromGetent(t *testing.T) { //nolint:paralleltest
 		})
 	}
 }
+
+func TestNetworkBytes(t *testing.T) {
```

Comment:
Do you think we can add a test for bad input there? Like, empty strings, characters instead of numbers etc. I know it's unlikely, but just to be 100% certain we won't panic.

Reply:
Good idea!